src/components/StarRating/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactComponent as Star } from 'assets/svg/star.svg';
+import { useState } from 'react';
+import styles from './StarRating.module.scss';
+
+interface StarRatingProps {
+  rating: () => void;
+}
+
+function StarContainer({ rating }: StarRatingProps) {
```

Comment:
props 의 네이밍이 컴포넌트와 달라야 할 이유가 있을까요...? 간단히 Props 로도 충분해보여요

```suggestion
interface Props {
  rating: () => void;
}

function StarContainer({ rating }: Props) {
```

Reply:
아 네이밍을 최대한 신경써 봤는데 그냥 Props로 줘도 괜찮군요..! 배워갑니다!

Reply:
외부에 노출시킨다면 보다 추상적인 props 네이밍을 고민해볼 수 있겠지만 위 케이스는 그렇지 않은것 같네요 😄